iphone/Classes/KrollBridge.m

```diff
@@ -818,7 +818,8 @@ - (TiModule *)loadTopLevelNativeModule:(TiModule *)module withPath:(NSString *)p
 	}
 
 	NSString* contents = [[[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding] autorelease];
-	KrollWrapper* wrapper = (id) [self loadJavascriptText:contents fromFile:path withContext:kroll];
+	NSURL *url_ = [TiHost resourceBasedURL:path baseURL:NULL];
+	KrollWrapper *wrapper = (id) [self loadCommonJSModule:contents withSourceURL:url_];
```

Comment:
It shouldn't have an impact. Specifically this is a case where we are loading javascript we received as a string and turning it into the exported object, then combining that with the previously loaded 'native' module. The native module with the layered commonjs exports is what should get cached later.